website/addons/evernote/static/evernoteLogActionList.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Added the file https://github.com/rdhyee/osf.io/commit/48eb9f336042bbb41124827d87aa4f8bcd138996